lib/windows/app/index.js

```diff
@@ -61,6 +61,18 @@ Module._load = function load(path) { // eslint-disable-line no-underscore-dangle
             return ReactDOM;
         case 'react-redux':
             return ReactRedux;
+        case 'nrfconnect/logger':
+            return api.logger;
+        case 'nrfconnect/pc-ble-driver-js':
+            return api.bleDriver;
```

Comment:
Normally, when using the pc-ble-driver-js library, the developer has to use it like this:

```
import { api } from 'pc-ble-driver-js';

const AdapterFactory = api.AdapterFactory;
```

In nRF Connect, we have skipped the `api` part:

```
import { AdapterFactory } from 'pc-ble-driver-js';
```

This is convenient, but the drawback is that we cannot refer directly to the pc-ble-driver-js API documentation. We have to point out that `nrfconnect/pc-ble-driver-js` is slightly different from `pc-ble-driver-js`. Should we change `nrfconnect/pc-ble-driver-js` to include the `api` part, so that we avoid confusion?